src/pages/BO/dashboard/index.ts

```diff
@@ -0,0 +1,11 @@
+import semver from "semver";
+import BOBasePage from "../BOBasePage";
+
+let dashboardPage: BOBasePage;
+if (semver.gte(process.env.PS_VERSION as string, '8.0.0')) {
```

Comment:
We need to Have a solution if we want 2 versions at the same time (ex: for Autoupgrade)

Reply:
In Autoupgrade, you will never have to use two versions at the same moment.

Reply:
You can use import from versions directory